<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggingWindow</class>
 <widget class="QMainWindow" name="LoggingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="importButton">
          <property name="text">
           <string>Import Data From Timetable Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableWidget" name="logPlanTable">
           <column>
            <property name="text">
             <string>Class</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Next Log</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Status</string>
            </property>
           </column>
          </widget>
          <widget class="QTableWidget" name="logDataTable">
           <column>
            <property name="text">
             <string>Date Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Air Quality</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Î” Air Quality</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Class</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Year</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Boys</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Girls</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="currentTimeLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00:00 00/00/00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="getTimeButton">
             <property name="text">
              <string>Check Current Time</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="showHideParentButton">
           <property name="text">
            <string>Hide Timetable Input Window</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Current Week:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="week1Button">
             <property name="text">
              <string>Week 1</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="week2Button">
             <property name="text">
              <string>Week 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="startLoggingButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start Logging</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nextLogLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Next Log:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="enableBackups">
             <property name="text">
              <string>Backup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="overwriteBackups">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Overwrite Old Backups</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="backupLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>New Logs/Backup:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="logsPerBackup">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadBackupButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Load Backup</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTabWidget" name="graphTabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Graph Wizard</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Graph Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="graphNameText"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Graph X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="xValueBox">
              <item>
               <property name="text">
                <string>Time</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Air Quality</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Air Quality Per Child</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Î” Air Quality</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Year</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Boys</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Girls</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="xInverted">
              <property name="text">
               <string>Inverted</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Graph Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="yValueBox">
              <item>
               <property name="text">
                <string>Time</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Air Quality</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Air Quality Per Child</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Î” Air Quality</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Year</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Boys</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Girls</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="yInverted">
              <property name="text">
               <string>Inverted</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="addGraphButton">
            <property name="text">
             <string>Add Scatter Graph</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
